test/Microsoft.AspNetCore.Mvc.Core.Test/Filters/FilterCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Testing;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Filters
+{
+    public class FilterCollectionExtensionsTests
```

Comment:
@rynowak do you want to duplicate the tests and use the generic overloads or change the existing tests to use the generic overload?